### ISSUE TITLE
Migrate from .net framework to .net core

### ISSUE THREAD

**Issue by sajjadarashhh** [dnnsoftware/Dnn.Platform#5732]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of problem

When will be dnn migrate from thats deprecated platform to dotnet core?

### Description of solution

I Dont have idea

### Do you be plan to contribute code for this enhancement?

- [X] Yes

### Would you be interested in sponsoring this enhancement?

- [X] Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by david-poindexter** [comment]
@sajjadarashhh please see the following two blog posts.  They still apply.
* https://dnncommunity.org/blogs/Post/7625/The-Technical-Future-of-DNN
* https://dnncommunity.org/blogs/Post/7626/Strategy-To-Know-Thyself

**Comment by matthewblott** [comment]
@david-poindexter [this](https://github.com/twsouthwick/systemweb-adapters-ui/issues/5) question I posted might be of interest to you, along with the project itself.

**Comment by mitchelsellers** [comment]
@matthewblott That initative looks promising, but we depend heavily on the out-of-scope items that they are not even trying to tackle so it would only be a minimal gain, and the complete recompilation requirement would be a show-stopper for any upgrade installation.

I appreciate the callout to that specific project though.

**Comment by david-poindexter** [comment]
Thanks @matthewblott

**Comment by sajjadarashhh** [comment]
Afterall when dnn migrate from that obsolete framework to .net core that give very very bigger than now 
And many contributers from main .net repository help this cms to give more perofrmance and take bigger and more useful than now and used blazor and hybrid applications that can be knocked out market share

**Comment by twsouthwick** [comment]
@mitchelsellers what are the out-of-scope items in that project? I'm the owner of it and thinking about spending some more time there. The need to recompile was something I was discussing with @birojnayak who's joining it - I have ideas on how to support that, just not sure how necessary it is.  I've created a new issue https://github.com/twsouthwick/systemweb-adapters-ui/issues/8 to track that - anything else you see as required would be helpful.

**Comment by mitchelsellers** [comment]
@twsouthwick maybe we can jump on a zoom sometime soon?

**Comment by david-poindexter** [comment]
> @twsouthwick maybe we can jump on a zoom sometime soon?

@mitchelsellers and @twsouthwick - I am happy to join if that happens.

**Comment by valadas** [comment]
Me too

**Comment by birojnayak** [comment]
@mitchelsellers apart from what @twsouthwick mentioned what are other 

> @mitchelsellers what are the out-of-scope items in that project? I'm the owner of it and thinking about spending some more time there. The need to recompile was something I was discussing with @birojnayak who's joining it - I have ideas on how to support that, just not sure how necessary it is. I've created a new issue [twsouthwick/systemweb-adapters-ui#8](https://github.com/twsouthwick/systemweb-adapters-ui/issues/8) to track that - anything else you see as required would be helpful.

@mitchelsellers feel free to create issues in the https://github.com/twsouthwick/systemweb-adapters-ui if you think there are blockers moving your project to .net core which should be in the webform scope. In general happy to see what other items apart from webform needs to be in .net core to unblock this project moving to .net core.Having a list would help.
